i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,59 @@
+from torch import nn
+
+
+class ConformerConvolutionV1(nn.Module):
+    """
+    Conformer convolution module.
+    see also: https://github.com/espnet/espnet/blob/713e784c0815ebba2053131307db5f00af5159ea/espnet/nets/pytorch_backend/conformer/convolution.py#L13
+    """
+
+    def __init__(self, channels, kernel_size, dropout=0.1):
```

Comment:
In RETURNN-common, we had `norm` as an additional (required) option here, to make it configurable.

There was no default for `norm`. Rather, the default was handled via the outer `ConformerEncoderLayer`. The default was `BatchNorm`.